pkg/util/kubeobjects/secret/secret_test.go

```diff
@@ -290,3 +309,55 @@ func TestSecretBuilder(t *testing.T) {
 		assert.True(t, found)
 	})
 }
+
+func TestCreateOrUpdate(t *testing.T) {
+	fakeClient := fake.NewClient()
+	fakeClient.Create(context.TODO(), testSecret.DeepCopy())
```

Comment:
just a minor detail: from what I have read `context.Background()` is mostly used for unit testing as the standard way to define an empty context. And here there is no TODO in my opinion, but just a minor nitpick (was wondering myself what to use in unit-tests :))

Reply:
True, thank you!
[Fixed here](https://github.com/Dynatrace/dynatrace-operator/pull/2594/commits/c9b9ab70719dae09169aa91704cc8c026ac18cfd)